--- <main>/Assets/Settings/FrontRooms_URP_Renderer.asset	2026-10-03 18:55:07
+++ <clone>/Assets/Settings/FrontRooms_URP_Renderer.asset	2026-10-07 16:54:50
@@ -93,5 +93,5 @@
   m_Script: {fileID: 11500000, guid: 8c7c221f6b7b4c88b8c10d39e5c1f5bd, type: 3}
   m_Name: FrontRooms Metal Glass RT
   m_EditorClassIdentifier: Assembly-CSharp::FrontRoomsMetalGlassRTRendererFeature
-  m_Active: 1
+  m_Active: 0
   prepassShader: {fileID: 4800000, guid: 9e07ae8a4fd2140a88b08079efa47be5, type: 3}
