--- <main>/Assets/Settings/FrontRooms_URP.asset	2026-10-04 21:31:02
+++ <clone>/Assets/Settings/FrontRooms_URP.asset	2026-10-07 18:50:07
@@ -51,8 +51,8 @@
   m_AdditionalLightsShadowResolutionTierLow: 256
   m_AdditionalLightsShadowResolutionTierMedium: 512
   m_AdditionalLightsShadowResolutionTierHigh: 1024
-  m_ReflectionProbeBlending: 0
-  m_ReflectionProbeBoxProjection: 0
+  m_ReflectionProbeBlending: 1
+  m_ReflectionProbeBoxProjection: 1
   m_ReflectionProbeAtlas: 1
   m_ShadowDistance: 40
   m_ShadowCascadeCount: 2
